File: app/src/main/java/com/cna/mineru/cna/DTO/ClassData.java
package com.cna.mineru.cna.DTO;

public class ClassData {
    public int ClassId;
    public int Term;
    public String Title;
    public int Tag;
    public int SubTag;

    public ClassData(int ClassId, int Term, String Title, int Tag, int SubTag){
        this.ClassId = ClassId;
        this.Term = Term;
        this.Title = Title;
        this.SubTag = SubTag;
        this.Tag = Tag;
    }

    public ClassData(int Tag, String Title){
        this.Title = Title;
        this.Tag = Tag;
    }
}

File: app/src/main/java/com/cna/mineru/cna/DTO/ClassData.kt
package com.cna.mineru.cna.DTO

class ClassData {
    var ClassId: Int = 0
    var Term: Int = 0
    var Title: String
    var Tag: Int = 0
    var SubTag: Int = 0

    constructor(ClassId: Int, Term: Int, Title: String, Tag: Int, SubTag: Int) {
        this.ClassId = ClassId
        this.Term = Term
        this.Title = Title
        this.SubTag = SubTag
        this.Tag = Tag
    }

    constructor(Tag: Int, Title: String) {
        this.Title = Title
        this.Tag = Tag
    }
}
